### PR TITLE
Add department as course on view, fix url on workspace, fix url for contact

### PR DESCRIPTION
**Pull request by jahnaviraj**

### CODE REVIEW

File: desk/src/components/desk/ticket/InfoPanel.vue [hunk 1]
```diff
@@ -408,6 +418,13 @@ export default {
 				).slice(0, 40)
 			}
 		},
+		contactDepartment() {
+			console.log(this.ticket.contact.department, "ticket contact")
```
Comment (ashishnpbs) [inline desk/src/components/desk/ticket/InfoPanel.vue:422]: remove log statement

File: frappedesk/setup/install.py [hunk 2]
```diff
@@ -15,11 +16,21 @@ def after_install():
 	update_agent_role_permissions()
 	add_default_assignment_rule()
 
+def set_login_page():
+	website_settings = frappe.get_doc("Website Settings")
+
```
Comment (ashishnpbs) [inline frappedesk/setup/install.py:21]: will it throw error if the doc is not there

File: frappedesk/setup/install.py [hunk 3]
```diff
@@ -15,11 +16,21 @@ def after_install():
...
+	if not website_settings.home_page:
+		website_settings.app_name = "Moodle Helpdesk"
+		website_settings.app_logo = "https://www.col.org/wp-content/uploads/2021/08/col-logo-stacked-color-1.png"
```
Comment (ashishnpbs) [inline frappedesk/setup/install.py:24]: instead of hardcoding urls - is there any config file where we can put it

**Review by ashishnpbs — CHANGES_REQUESTED** [review]

File: frappedesk/setup/install.py (hunk 2 above)
Comment (jahnaviraj) [reply frappedesk/setup/install.py:21]: doc is by default created by frappe, so when we install frappedesk it will be there 100%

**Pull request merged**